src/measure_theory/integral/lebesgue.lean

```diff
@@ -1446,6 +1461,21 @@ begin
   exact indicator_apply_le id
 end
 
+lemma lintegral_eq_top_of_measure_eq_top_pos {f : α → ℝ≥0∞} (hf : measurable f)
+  (hμf : 0 < μ {x | f x = ∞}) : ∫⁻ x, f x ∂μ = ∞ :=
+eq_top_iff.mpr $
+calc ∞ = ∞ * μ {x | ∞ ≤ f x} : by simp [mul_eq_top, hμf.ne.symm]
+   ... ≤ ∫⁻ x, f x ∂μ : mul_meas_ge_le_lintegral hf ∞
+
+lemma eventually_lt_top_of_lintegral_ne_top {f : α → ℝ≥0∞} (hf : measurable f)
+  (hμf : ∫⁻ x, f x ∂μ ≠ ∞) : ∀ᵐ x ∂μ, f x < ∞ :=
```

Comment:
I just realized that this lemma is almost identical to `ae_lt_top`, in lebesgue.lean:
```
ae_lt_top {f : α → ℝ≥0∞} (hf : measurable f) (h2f : ∫⁻ x, f x ∂μ < ∞) :
  ∀ᵐ x ∂μ, f x < ∞
```

Reply:
good catch, I'll remove `eventually_lt_top_of_lintegral_ne_top` then!